app/models/project.rb

```diff
@@ -932,24 +930,6 @@ def archive!
     update_attribute :status, STATUS_ARCHIVED
   end
 
-  def required_storage
-    Rails.cache.fetch("project##{id}/project_storage",
-                      expires_in: self.class.project_storage_expires_in) do
-      count_for(:required_project_storage).first
-    end
-  end
-
-  def self.total_projects_size
-    Rails.cache.fetch('projects/total_projects_size',
-                      expires_in: project_storage_expires_in) do
-      Project.all.map { |p| p.required_storage[:total] }.sum
-    end
-  end
-
-  def self.project_storage_expires_in
-    Setting.project_storage_cache_minutes.to_i.minutes
```

Comment:
I have now [removed the projects-related setting](https://github.com/opf/openproject/pull/3368/files#diff-646ad6b10917e4385cbcc8c8524e24a2L281), the repository one is still in use.